packages/moonstone/Input/Input.js

```diff
@@ -247,7 +263,7 @@ const InputBase = kind({
 			const title = (value == null || value === '') ? placeholder : '';
 			return calcAriaLabel(title, type, value);
 		},
-		className: ({focused, invalid, small, styler}) => styler.append({focused, invalid, small}),
+		className: ({focused, invalid, size, small, styler}) => styler.append({focused, invalid}, !size && small ? 'small' : size || 'medium'),
```

Comment:
If we include `size` in the `append`, will it be easier to update later?

```
styler.append({focused, invalid}, size, !size && (small ? 'small' : 'medium'))
```

Although, should we specify a default for a prop and not enforce it through `defaultProps`?  Could we implement this so it took that into account?